### ISSUE TITLE
Update the npm package?

### ISSUE THREAD

**Issue by s1nistr4** [fluent-ffmpeg/node-fluent-ffmpeg#1255]
The NPM package hasn't been updated in 7 years and there contains a lot of ancient ES3 code , and the github repo has a lot of changes made over time. Could we get an update to it

**Comment by princefishthrower** [comment]
Yep, and there is an ugly issue when trying to package anything ES that uses this library (at least with rollup, but I guess the root problem lies here)

```
[!] (plugin rpt2) RollupError: [plugin rpt2] ./lib-cov/fluent-ffmpeg?commonjs-external: Could not resolve "./lib-cov/fluent-ffmpeg" from "./lib-cov/fluent-ffmpeg?commonjs-external"
```

Going to snoop around the other 300+ (!) open issues here to see if there is a solution....

**Comment by Gahaghan** [comment]
I'm glad to see this project waking up, but I have a minor warning.

The update from from version 2.1.2 to 2.1.3 has a small breaking change.  Version 2.1.2 works with Node 16.  Version 2.13 requires Node 18+.   I understand that Node 16 has reached end of life, but after 7 years perhaps there should have been a version bump to 2.2.0.

I had to change my package.json dependencies setting from

  "dependencies": {
    "fluent-ffmpeg": "^2.1.2"
  }

TO

  "dependencies": {
    "fluent-ffmpeg": "2.1.2"
  }
